src/app/components/character-count/index.html

```diff
@@ -0,0 +1,284 @@
+<sky-tabbed-demo-page
+  pageTitle="Character Count"
+>
+
+  <sky-demo-page-summary>
+  The character count component is an extension for text input fields with character limits.
+  </sky-demo-page-summary>
+
+  <div
+    class="sky-margin-stacked-content-section"
+  >
+    <sky-demo-page-example>
+      <sky-character-count-demo>
```

Comment:
The demo includes name fields with a character limits of 10 characters. Does that seem like a realistic example? I don't think so. Is there a more realistic example that we can use here than first names?

Reply:
Also, is there a reason that the reactive form example includes a default value in the field and that the template-driven example does not?

Reply:
No real reason one has the default when the other doesn't other than to show that initializing the field with a value will set the counter correctly. Would you say they should both have a default, or neither?

Reply:
I don't have a strong preference. I usually opt for consistency, but in this case, it sounds like it makes sense a default in one and not in the other to highlight the difference in behavior when initializing. I just was checking to make sure this was a conscious decision and not just an accidental inconsistency.

As for the first question? Does anyone have ideas for a more realistic example than a Name field?

Reply:
Just double-checking to see if the Name example is a legit use case. If so, we can leave it as is. If not, maybe we can tweak it to something more realistic.